src/runtime_src/core/include/experimental/xrt_pscontext.h

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 Advanced Micro Devices, Inc. All rights reserved.
+#ifndef XRT_PSCONTEXT_H_
+#define XRT_PSCONTEXT_H_
+
+#include "xrt/detail/config.h"
+#include "xrt/detail/pimpl.h"
+#include "xclhal2.h"
+
+#ifdef __cplusplus
+#include <memory>
+
+/*
+ * PS Context Data Structure to be derived from for user's xrtHandle class
```

Comment:
This comment doesn't reflect usage.  The comment is for xrt::pscontext presumably, but that class is not derived from user's xrt_handle class.   Next line of comment is correct though.

Also can we use xrt_handle rather than camelCase.  There is no camelCase in xrt exposed APIs.